CHANGELOG.md

```diff
@@ -21,3 +21,7 @@ WIP
 * Send server and protocol versions along with handshake response [#103](https://github.com/code-mancers/rbkit/pull/103)
 * Split object dump into smaller messages [#77](https://github.com/code-mancers/rbkit/pull/77)
 * Fix bug with handshake broken because of Rbkit::VERSION not being available.
+* Make REQ and PUB ports configurable using ENV variables
+* Always bind to 127.0.0.1
+* Add CPU Sampling profiler and cpu_sample message type. Bump up protocol version to v2.1
+* Use C89 std for compiling rbkit
```

Comment:
In our dev boxes C99 is what gets used for compilation, but in travis, it uses C89 and gives compilation warnings. I thought we should stick to one std , so I did some research to see which one to use. Looks like Ruby-core officially supports only C89 std, also C89 is the most widely implemented standard.